public/src/inverted-index.js

```diff
@@ -92,63 +101,57 @@ class Index {
     return check;
   }
 
-/**
- * @method getIndex
- *
- * Method return index of files in indices object
- *
- * @param {string} name
- * @returns {object}
- */
-  getIndex(name) {
-    if (name && typeof name === 'string') {
-      return this.indices[name];
-    }
-    return this.indices;
-  }
+  /**
+  * @method getIndex
+  *
+  * Method return index of files in indices object
+  *
+  * @param {string} name
+  * @returns {object}
+  */
+  getIndex(name) { return name && typeof name === 'string' ? this.indices[name] : this.indices; }
 
-/**
- * @method searchFeedback
- *
- * Takes terms of array and fetch result of each token.
- *
- * @param {array} termsArray
- * @param {object} file
- * @returns {object} searchResults
- */
-  searchFeedback(termsArray, file) {
+  /**
+  * @method doSearch
+  *
+  * Takes terms of array and fetch result of each token.
+  *
+  * @param {array} termsArray
+  * @param {object} file
+  * @returns {object} searchResults
+  */
+  doSearch(termsArray, fileIndex) {
     const searchResults = {};
     termsArray.forEach((term, index) => {
-      if (file.hasOwnProperty(term)) {
-        searchResults[termsArray[index]] = Array.from(file[term]);
+      if (fileIndex.hasOwnProperty(term)) {
+        searchResults[termsArray[index]] = Array.from(fileIndex[term]);
       } else {
         searchResults[term] = [];
       }
     });
     return searchResults;
   }
 
-/**
- * @method searchIndex
- *
- * Looks for search terms in created index
- *
- * @param {object} currentFile
- * @param {array} searchTerm
- * @returns {object} searchResults
- */
+  /**
+  * @method searchIndex
+  *
+  * Looks for search terms in created index
+  *
+  * @param {object} currentFile
+  * @param {array} searchTerm
+  * @returns {object} searchResults
+  */
   searchIndex(currentFile, ...searchTerm) {
```

Comment:
### Nitpick

The second argument should be pluralized i.e be `pluralized` because of the `...` it's a collection of terms